k8s/elasticsearch.yaml

```diff
@@ -92,6 +141,14 @@ spec:
         requests:
           storage: 25Gi
```

Comment:
Use powers of 2 when specifying a volume size, as volumes are billed as such. Either use 16Gi or 32Gi.